## FastQC
Multiple 	https://www.bioinformatics.babraham.ac.uk/projects/fastqc/ 	This is a common tool used to check reads quality from different sequencing technologies such as Illumina, 454, and PacBio.

## BWA
Short & Long reads 	https://sourceforge.net/projects/bio-bwa/files/ 	This is a command line tool. Mostly known for lightweight run and accurate sequence alignment.

## MiniMap2

Long reads 	https://github.com/lh3/minimap2 	This command line tool is designed to handle PacBio & Oxford Nanopore and reads with 15% error rate.

## bwa-mem2 

Short & Long reads 	https://github.com/bwa-mem2/bwa-mem2 	This command line tool is a performance optimized development, and drop-in replacement, of the BWA program.[18]

## bwa-meme 

Short & Long reads 	https://github.com/kaist-ina/BWA-MEME 	This command line tool is a performance optimized development, and drop-in replacement, of the bwa-mem2 algorithm utilizing Machine Learning.[19]

## LoReTTA 

Long reads 
https://github.com/salvocamiolo/LoReTTA/releases/tag/v0.1 
This tool is designed to assemble (reference-guided) viral genomes at a greater accuracy using PacBio CCS reads.

## SPAdes 

Short & Long reads 
https://ablab.github.io/spades/index.html	
This is an assembly tool that runs on the command line.

## Samtools 	

Alignment analysis 
https://samtools.github.io 	
This is useful post-assembly. It can generate different statistics and perform multiple filtering steps to the alignment file. 